cves/kernel/CVE-2019-15807.yml

```diff
@@ -151,10 +145,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Jian Luo reported that the Serial Attached SCSI library (libsas) did not
+    correctly handle the failure to discover devices beyond a SAS expander. This
+    could lead to a resource leak and crash (BUG). The security impact of this
+    is unclear.
```

Comment:
Is it STILL unclear or was it just unclear at the time of the bug report? If it's still unclear maybe you could take a best guess?